.github/actions/label-and-move-released-issues-v1/action.yml

```diff
@@ -17,7 +17,7 @@ inputs:
   column-name:
     description: 'Name of column to move to'
     required: false
-    default: 'released'
```

Comment:
Can you update this in the `README` as well and clarify there that leaving it unspecified will result in skipping the column movement?